src/checkfnis.cpp

```diff
@@ -1,29 +1,44 @@
 #include "checkfnis.h"
 
 #include "iplugingame.h"
+#include "pluginsetting.h"
 #include "report.h"
-#include <scopeguard.h>

```

Comment:
This isn't right? scopeguard.h and questionboxmemory.h aren't part of this repository.